README.md

```diff
@@ -113,14 +152,15 @@ vars:
       alias: one_two_three
   stripe__plan_metadata:
     - name: rename
-    - alias: renamed_field
+      alias: renamed_field
+  stripe__price_metadata:
```

Comment:
Updating to reflect the changed variable name.
```suggestion
  stripe__price_plan_metadata: ## Used for both Price and Plan sources
```

Reply:
Additionally, maybe use a different variable as an example below for how you can use the old array version of the variable. Otherwise, update the variable to be `stripe__price_plan_metadata`
![image](https://user-images.githubusercontent.com/74217849/215216681-8b3c130d-7299-4440-a347-99b18588f9ce.png)

Reply:
Oh I see what you mean-- yeah just updated it to use `stripe__subscription_metadata` arbitrarily